requirements.txt

```diff
@@ -3,4 +3,4 @@ selenium==3.4.3
 keras==2.1.1
 tensorflow==1.3.0
 h5py==2.7.1
-requests==2.18.4
+requests==2.18.4
```

Comment:
Can you remove the changes from this file?

Reply:
what is changed in this line?